src/ModalFooter.js

```diff
@@ -21,7 +21,7 @@ class ModalFooter extends React.Component<ModalFooterProps, {}> {
                     </Button>
                 }
                 <Button className="ModalFooter__doneButton" onClick={this.props.onClickDone}>
```

Comment:
Please also rename the callback property to `onClickSave`, and any related handlers to `handleClickSave` or such.

Reply:
Actually. I'm having some different thoughts about this now.

Reply:
I think we want the `ModalFooter` to be more flexible than it currently is. Right now we have:

* Always a 'Done' button, with callback `onClickDone` which this PR gives the label "Save"
* And an optional 'Cancel' button, with callback `onClickCancel`

Looking at the Figma designs, I think we will need a few different variations:

* Cancel, Save (Available Actions, Background, Keyboard Shortcuts, Sound Options, Themes)
* Cancel (Share Link, though maybe this should be "Done" rather than "Cancel"?)
* Cancel, Delete (Delete Program)
* Cancel, Connect (Robot Connections, though I think that we cannot make this UI due to limitations in the functionality of the web Bluetooth API)

So mostly we have (Cancel, Save), but we also have (Cancel, Delete) and possibly Cancel only (though that might be better Done only).

Reply:
A possibility would be to change the `ModalFooter` properties to take a list of (label, callback) pairs. That way the footer could display any number of buttons, with different semantics. Something like:

```
<ModalFooter
    buttons={[
        {
            label: this.props.intl.formatMessage({ id: 'CancelButton' }),
            callback: this.handleClickCancel
        },
        {
            label: this.props.intl.formatMessage({ id: 'SaveButton' }),
            callback: this.handleClickSave
        }
    ]}
/>
```